admin/context_test.go

```diff
@@ -0,0 +1,58 @@
+package admin_test
```

Comment:
* What CI service have you looked at (I saw you signed up for a couple)? 
* Did you pick one?
* How about `cbr.theplant-dev.com` or `ci.theplant-dev.com`?
* Do you have any other goals for "working on CI"?